back/transcendence_django/back_game/game_geometry/vector.py

```diff
@@ -5,6 +5,10 @@
 
 class Vector(Position):
 
+    def __init__(self, x: float, y: float):
```

Comment:
Position ctor should get called by default when supplied with x and y args? No need to redefine?